tailwind.config.js

```diff
@@ -5,7 +5,22 @@ module.exports = {
   theme: {
     extend: {
       colors: {
-        primary: '#FF6363'
+        primary: {
+          dark: '#d73636',
+          DEFAULT: '#d73636'
+        },
+        background: {
+          dark: '#23343b',
+          DEFAULT: '#ffffff'
+        },
+        text: {
+          dark: '#000000',
```

Comment:
Dark Text should be #212121

Reply:
Text dark is FFF. DARK is used for dark theme. default will be 212121